modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -227,6 +229,34 @@ public Object addSystemObjectRelationshipMappingTableValues(
 			systemObjectDefinitionManager);
 	}
 
+	@Override
+	public ObjectEntry copyVersionedObjectEntry(
+			DTOConverterContext dtoConverterContext, Long objectEntryId,
+			ObjectDefinition objectDefinition, Integer version)
+		throws Exception {
```

Comment:
Besides deciding whether to rename the method, can you sort the parameters?

This class does not follow service.xml; instead, it is ordered alphabetically (as seen in the [example](https://github.com/liferay/liferay-portal/blob/b6cfe9e03336186cb5fed40aab117f75c0631d64/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java#L593)).

Reply:
Done